contracts/token/SyndicateERC20.sol

```diff
@@ -300,8 +300,8 @@ contract SyndicateERC20 is AccessControl {
   uint32 public constant ROLE_WHITE_LISTED_SPENDER = 0x0010_0000;
 
   /**
-   * @notice White listed treasury can swap sSYN to receive SYNR
-   * @dev Role ROLE_TREASURY can get SYNR in exchange for sSYN
+   * @notice White listed treasury can swap sSYNR to receive SYNR
+   * @dev Role ROLE_TREASURY can get SYNR in exchange for sSYNR
    */
```

Comment:
This contract should never change, since it is already deployed. But since the change is only in the comments, it is fine.